src/client/components/ShoppingCart/ShoppingCartImage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ShoppingCart.styles.css';
+
```

Comment:
Try import products from offerProductsList.component rather than hardcoding for discount products.